### DISCUSSION TITLE
Display WeekDay with 3 characters instread of only 2.

### DISCUSSION THREAD

**Discussion by shehzi-khan** [gpbl/react-day-picker#1232] [General]
I want to display WeekDay with 3 characters instread of only 2.
Is there a straight forward way like `<DatePicker useWeekdaysShort={true} />`

**Comment by oudoulj** [comment]
You can use `weekdaysLong` with your own translation strings and therefore choose whatever length you want 😉 
https://react-day-picker.js.org/docs/localization

```
const WEEKDAYS_LONG = [
  'Domenica',
  'Lunedì',
  'Martedì',
  'Mercoledì',
  'Giovedì',
  'Venerdì',
  'Sabato',
];

<DayPicker
  locale="it"
  weekdaysLong={WEEKDAYS_LONG}
/>
```
